bubblewrap.c

```diff
@@ -439,6 +439,19 @@ has_caps (void)
   return data[0].permitted != 0 || data[1].permitted != 0;
 }
 
+static void
+drop_cap_bounding_set (void)
+{
+  unsigned long cap;
+
+  for (cap = 0; cap <= 63; cap++)
```

Comment:
Where did this 63 value come from?  A quick glance at the systemd source (again my first go-to reference) is they have a `cap_last_cap()` function which does all sorts of gymnastics.  But on the other hand it wouldn't be that bad to copy.